apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -22,8 +134,15 @@
   padding: 20px;
 }
 
+.webcam-container {
+  background-color: white;
```

Comment:
Properties should be ordered margin-bottom, background-color

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)